core/includes/theme.inc

```diff
@@ -931,10 +931,10 @@ function backdrop_find_base_themes($themes, $key, $used_keys = array()) {
  * @subsection sub_preprocess_templates Preprocessing for Template Files
  * If the implementation is a template file, several functions are called
  * before the template file is invoked, to modify the $variables array. These
- * fall into the "preprocessing" phase and the "processing" phase, and are
- * executed (if they exist), in the following order (note that in the following
- * list, HOOK indicates the theme hook name, MODULE indicates a module name,
- * THEME indicates a theme name, and ENGINE indicates a theme engine name):
+ * fall into the "preprocessing" phase, and are executed (if they exist) in the
+ * following order: (Note that in the following list, HOOK indicates the theme
+ * hook name, MODULE indicates a module name, THEME indicates a theme name, and
+ * ENGINE indicates a theme engine name.)
```

Comment:
The formatting of the period inside the brackets and the colon removed were proper before this change. They should be restored:
```suggestion
 * following order (note that in the following list, HOOK indicates the theme
 * hook name, MODULE indicates a module name, THEME indicates a theme name, and
 * ENGINE indicates a theme engine name):
```